project/src/main/java/com/google/sps/data/Restaurant.java

```diff
@@ -0,0 +1,23 @@
+package com.google.sps.data;
+
+import com.google.auto.value.AutoValue;
+import java.util.List;
+import java.util.Map;
+
+@AutoValue
```

Comment:
optional further reading: apparently there's ways to add support to create autovalue objects using Gson: https://github.com/rharter/auto-value-gson by adding TypeAdapteFactories - probably a bit much to set up etc for this (looks a lot cleaner already with converting to a pure-value holder class) but just a bit of further explorations :D